### PR TITLE
Creating workspace id in section

### PR DESCRIPTION
**Pull request by arpita-20**

### CODE REVIEW

File: prisma/migrations/20221004100256_workspace_queries/migration.sql [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+DECLARE 
+  userIdVar varchar;
+  firstNameVar varchar;
+  roleIDVar varchar;
+  random_id varchar;
+  userWorkspaceId varchar;
+  userList record;
+BEGIN
+    FOR userList in SELECT *  FROM "User" LOOP
+      random_id=gen_random_uuid();
```
Comment (richa-copods) [inline prisma/migrations/20221004100256_workspace_queries/migration.sql:13]: Please rename it to workspaceId

File: prisma/migrations/20221004100256_workspace_queries/migration.sql [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              workspaceIdCopods=workspaceList."id";
+            end if;  
+        end loop;    
+      END IF;
+    END LOOP; 
+    for userList in select * from "User" LOOP
+      userWorkspaceId=gen_random_uuid();  
+      PERFORM * from "User" where userList."email"='careers@copods.co';
+      if not found then
+        userIdVar=userList."id";
```
Comment (richa-copods) [inline prisma/migrations/20221004100256_workspace_queries/migration.sql:51]: No need to create these variables, values can be used directly

File: prisma/schema.prisma [hunk 3]
```diff
@@ -7,21 +7,53 @@ generator client {
...
+  updatedAt       DateTime          @updatedAt
+  Section         Section[]
+  Test            Test[]
+  CandidateResult CandidateResult[]
+
+  @@unique([name, createdById])
+}
+
+model UserWorkspace {
+  id          String    @id @default(cuid())
```
Comment (richa-copods) [inline prisma/schema.prisma:26]: As we are using uuid so please update this

File: prisma/schema.prisma [hunk 4]
```diff
@@ -7,21 +7,53 @@ generator client {
   provider = "prisma-client-js"
 }
 
+model Workspace {
+  id              String            @id @default(cuid())
```
Comment (richa-copods) [inline prisma/schema.prisma:11]: We are going ahead with uuid right? Please update this

**Review by richa-copods — CHANGES_REQUESTED** [review]

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**